src/evm_context.rs

```diff
@@ -132,12 +134,24 @@ impl MemoryRange {
     }
 }
 
-pub fn set_reverted(ssa: &mut SSATracker) -> SMTStatement {
-    assign_variable_if_executing_regularly(ssa, &context().revert_flag.identifier, 1.into())
+pub fn revert(input: MemoryRange, ssa: &mut SSATracker) -> Vec<SMTStatement> {
+    let sig = smt::ite(
+        smt::eq(input.length.clone(), 0x24),
```

Comment:
Why not `bvuge`?

Reply:
I think it should even be `bvuge(length, 4)`

Reply:
Yea true